### ISSUE TITLE
[function converttoprettysymbol] : Assertion failed: ((unsigned int) currentchar < alphabet->mapsize-1)

### ISSUE THREAD

**Issue by BioWu** [genometools/genometools#858]
## Problem description

Assertion failed: ((unsigned int) currentchar < alphabet->mapsize-1), function converttoprettysymbol, file src/core/alphabet.c, line 754

## Exact command line call triggering the problem
I do not know which  step could trigger this problem, as the commands were applied by the LTRSIFT software. This might caused by the Genome file, just a suggestion.

```
#Using [ltrsift]
wget http://www.zbh.uni-hamburg.de/fileadmin/gi/LTRsift/ltrsift-1.0.2-Linux_x86_64-64bit.tar.gz

./run_ltrsift

#Genome used:
wget ftp://ftp.ncbi.nlm.nih.gov/genomes/all/GCF_000142985.2_Acyr_2.0/GCF_000142985.2_Acyr_2.0_genomic.fna.gz

#The LTR annotation

##LTR_finder and LTR_digest (results)

```

## Example minimal input triggering the problem
The Genome file download link is [Acyr_2.0](ftp://ftp.ncbi.nlm.nih.gov/genomes/all/GCF_000142985.2_Acyr_2.0/GCF_000142985.2_Acyr_2.0_genomic.fna.gz
).
[test.gff3.txt](https://github.com/genometools/genometools/files/995265/test.gff3.txt)

## What GenomeTools version are you reporting an issue for (as output by `gt -version`)?
gt (GenomeTools) 1.5.9
Copyright (c) 2003-2016 G. Gremme, S. Steinbiss, S. Kurtz, and CONTRIBUTORS
Copyright (c) 2003-2016 Center for Bioinformatics, University of Hamburg
See LICENSE file or http://genometools.org/license.html for license details.

Used compiler: cc (GCC) 4.8.5 20150623 (Red Hat 4.8.5-11)
Compile flags:  -g -Wall -Wunused-parameter -pipe -fPIC -Wpointer-arith -Wno-unknown-pragmas -O3 -Werror

## Did you compile GenomeTools from source? If so, please state the `make` parameters used.

```
make  64bit=yes cairo=no 
```
## What operating system (e.g. Ubuntu, Mac OS X), OS version (e.g. 15.10, 10.11) and platform (e.g. x86_64) are you using?

CentOS 7 x86_64

**Comment by BioWu** [comment]
The above mistake toke place at ```gt ltrclustering -force -psmall 30 -plarge 30 -o test.clster test.fa test.gff3```. It seems like that is not error of ```ltrsift```, instead, the function ```gt_ltr_cluster_stream_new```  called in ```project_wizard.c``` is failed. I did not know what had happened and how to solve this problem.

**Comment by BioWu** [comment]
This problem occurred when the function ```gt_alphabet_decode``` was called at regions contained the last base of  reference sequence. eg. such a region ```chr1    10    200    .    .    +``` was called when the length of ```chr1``` was ```200bp```.

**Comment by zhangbob159** [comment]
Has the problem solved?

**Comment by zhangbob159** [comment]
How to solve the problem---" ((unsigned int) currentchar < alphabet->mapsize-1), function converttoprettysymbol, file src/core/alphabet.c, line 759"?

**Comment by satta** [comment]
This should be addressed in #904.
